libbeat/processors/config.go

```diff
@@ -125,8 +125,10 @@ func extractInt(unk interface{}) (uint64, error) {
 func extractString(unk interface{}) (string, error) {
 	switch s := unk.(type) {
 	case string:
-		return string(s), nil
+		return s, nil
+	case *string:
```

Comment:
I was expecting that `ConvertToGenericEvent` should fix that after marshalling & unmarshalling a pointer to string, becomes string: https://github.com/elastic/beats/blob/master/libbeat/common/event.go#L70. But I assume it doesn't do that.
Ideally, it would be nice if we can restrict somehow the type of the fields in the event, but for now an warning should be enough.